config/locales/en/main.yml

```diff
@@ -1427,10 +1431,12 @@ en:
   operation:
     clear_confirm: "Are you sure you want to clear all completed operations?"
     server_error: "Server error. Please contact support."
+    no_attachment: "Operation does not have an attachment"
     description:
       user_import_operation_job: User import for %{mission_name} from %{file}
       option_set_import_operation_job: Option Set '%{name}' import for %{mission_name}
-    outcome_link_text: "link"
+      response_csv_export_operation_job: Response CSV export for %{user_email} in %{mission_name}
```

Comment:
Let's quote these three lines please.

Reply:
Also can we remove user_email and mission_name from all the above? Those are included in a more structured way and don't belong here. And can we add a count of the exported responses?

Reply:
@smoyth We don't know how many responses there will be until the worker starts running (it executes the database query).  This description text is used when enqueueing the job.